package.json

```diff
@@ -4,6 +4,18 @@
   "main": "lib/cjs/index.js",
   "types": "lib/cjs/index.d.ts",
   "module": "lib/esm/index.js",
+  "exports": {
+    ".": {
+      "types": "./esm/index.d.ts",
+      "import": "./esm/index.js",
+      "require": "./cjs/index.js"
+    },
+    "./*": {
+      "types": "./esm/*.d.ts",
+      "import": "./esm/*.js",
+      "require": "./cjs/*.js"
```

Comment:
can we remvoe the folder cherry picking now since this should cover all those cases? I've not tested it but it should work. maybe that counts as a breaking change tho

Reply:
I think we still need the cherry picked folders for backwards compat.  This TS feature was only avail in 4.7+

Reply:
Ah ok, it's been supported in bundlers and node generally for a lot longer, still, it's progress